src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFileUtil.java

```diff
@@ -48,13 +51,13 @@ of this software and associated documentation files (the "Software"), to deal
      * @throws InterruptedException
      * @throws AbortException       config file has not been found
      */
-    public static Map<ManagedFile, FilePath> provisionConfigFiles(List<ManagedFile> managedFiles, Run<?, ?> build, FilePath workspace, TaskListener listener, List<String> tempFiles) throws IOException, InterruptedException {
+    public static Map<ManagedFile, FilePath> provisionConfigFiles(List<ManagedFile> managedFiles, @Nullable EnvVars env, Run<?, ?> build, FilePath workspace, TaskListener listener, List<String> tempFiles) throws IOException, InterruptedException {
```

Comment:
The signature of this method can not be changed, I know of other plugins using it. Please add a second/new one (where one delegates to the other) and mark this one a deprecated.

Reply:
Updated.